packages/toggle/src/Toggle/Toggle.spec.tsx

```diff
@@ -1,40 +1,39 @@
 import React from 'react';
-import { act, fireEvent, render, screen } from '@testing-library/react';
+import { act, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { axe } from 'jest-axe';
 
+import { getLGToggleTestUtils } from '../utils';
 import Toggle from '..';
 
 const className = 'test-className';
 
 function renderToggle(props = {}) {
-  const utils = render(
+  const renderUtils = render(
     <>
       <label id="label" htmlFor="toggle">
         test
       </label>
-      <Toggle
-        aria-labelledby="label"
-        id="toggle"
-        data-testid="toggle-test-id"
-        {...props}
-      />
+      <Toggle aria-labelledby="label" id="toggle" {...props} />
     </>,
   );
-  const toggle = utils.getByRole('switch') as HTMLButtonElement;
 
-  return { ...utils, toggle };
+  const { elements, utils } = getLGToggleTestUtils();
+  const toggle = elements.getInput();
+
+  return { ...renderUtils, elements, utils, toggle };
```

Comment:
We should try and be consistent in these `renderXYZComponent` functions with whether we spread `...elements` and `...utils` or leave them as separate properties. We